themes/makedeb/static/js/theme.js

```diff
@@ -0,0 +1,47 @@
+initialThemeLoad();
+
+const toggleThemeBtn = document.getElementById("toggle-theme-button");
+toggleThemeBtn?.addEventListener("click", function () {
```

Comment:
I actually didn't know about this `?` stuff until now, apparently it's called [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)? If so, we should probably remove it so that errors are raised when elements don't exist (since they always should), right?